features/step_definitions/files_steps.rb

```diff
@@ -11,11 +16,21 @@
 
 
 
+Then(/^I (?:still|again) have my uncommitted file$/) do
+  verify_uncommitted_file file_name: @uncommitted_file_name
+end
+
+
 Then(/^(?:now I|I still) have the following committed files$/) do |files_data|
   files_data.diff! files_in_branches
 end
 
 
+Then(/^I don't have my uncommitted file$/) do
```

Comment:
Well it doesn't check that it's stashed away, so the definition would need to change to match. This is related to #480 's step, so what do you think of switching to `still stashed away` after that PR?